qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ReaderCloseTest.java

```diff
@@ -68,7 +68,7 @@ public void shouldDeleteCompactedSegmentsFilesAfterLeaderChange() {
             .getConfig()
             .getCluster()
             .getNodeId();
-    clusteringRule.forceClusterToHaveNewLeader(followerId);
+    clusteringRule.forceClusterToHaveNewLeader(1, followerId);
```

Comment:
:wrench: Maybe we rename the method to `forceClusterToHaveNewLeaderForPartition` and switch the order of the parameters, then it might be more clearer for which the different numbers stand for.

:thought_balloon: Besides that, I see no different usage of the parameter yet.

Reply:
:+1: Will rename and reorder parameters.